README.md

```diff
@@ -30,3 +30,7 @@ To install the dependency properly, issue from the project root:
 ## Documentation
 For more details and sample analysis please refer to the documentation located in `documentation` folder.
 For any questions and comments please contact the author on GitHub or [mail](mailto: marsik@ksi.mff.cuni.cz).
+
+## Troubleshooting
+1. You may get: `Exception in thread "main" java.lang.UnsupportedClassVersionError: org/harmony_analyser/application/HarmonyAnalyser : Unsupported major.minor version 52.0`. That means you tried to install a new version of JRE when you already have a previous one. Make sure you set up the default JRE properly. You can check the default Java version used on your UNIX system with: `sudo update-alternatives --config java`.
```

Comment:
Perfect idea to add troubleshooting, thanks for the PR!

Can we only change the `you tried to install a new version of JRE` to `you have a wrong version of JRE set up as your default JRE`? and `Make sure you have Java 8 installed and your JRE is set up properly`? The same error occurs when you simply don't have Java 8 because you only had Java 7 on your system.